pyvista/utilities/geometric_objects.py

```diff
@@ -1218,6 +1219,72 @@ def Circle(radius=0.5, resolution=100):
     return pyvista.wrap(pyvista.PolyData(points, cells))
 
 
+def Superquadric(center=(0., 0., 0.), scale=(1., 1., 1.), size=0.5,
+                 thetaroundness=1., phiroundness=1.,
+                 thetaresolution=16, phiresolution=16,
+                 toroidal=False, thickness=1/3):
```

Comment:
Default values taken from the `Set` functions in [vtkSuperquadricSource](https://vtk.org/doc/nightly/html/classvtkSuperquadricSource.html).